_data/finance.yml

```diff
@@ -17,6 +17,12 @@ websites:
       twitter: freshbooks
       tfa: No
 
+    - name: Geico
+      url: https://www.geico.com/
+      img: geico.png
+      twitter: geico
```

Comment:
Looks like they answer questions on `GEICO_Service`.
Change the Twitter handle please